lavalink/src/node.rs

```diff
@@ -488,6 +498,15 @@ impl Connection {
     }
 }
 
+impl Drop for Connection {
+    fn drop(&mut self) {
+        // Cleanup local players associated with the node
+        self.players
+            .players
+            .retain(|_, v| v.node().config() != &self.config);
```

Comment:
Does this have to compare all of the config?

Reply:
Changed to only compare the socket address instead.